contracts/openapi-stub.yaml

```
@@ -0,0 +1,379 @@
+components:
+  securitySchemes:
+    bearerAuth:
+      type: http
+      scheme: bearer
+      bearerFormat: JWT
+
+  schemas:
+    Organization:
+      type: object
+      properties:
+        id:        { type: string, format: uuid }
```

YAMLlint (1.38.0)

[error] too many spaces after colon

(colons)